client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -71,7 +75,13 @@ const StockListComponent: FC = () => {
       Cell: TooltipTextCell,
       width: 350,
     },
-    // TODO:: Add a column for the master list name
+    {
+      key: 'masterList',
+      label: 'label.master-list',
+      Cell: ChipTableCell,
```

Comment:
![Screenshot 2025-01-20 at 4 06 26 PM](https://github.com/user-attachments/assets/bc32d0a9-d04d-47ed-a16d-8081c0ee8968)

I think if we're going to show them like this, we should hide master list column by default. Item could be on a LOT of master lists....

Reply:
Yeah... but Mark wanted to show master list column since we're filtering by it now... the same thing will happen to program names too in patient list if programs is configured... I think we should just show default number of chips (will need issue) and show ellipsis or a more > button.. will need design